mtp_api/settings/base.py

```diff
@@ -73,23 +65,29 @@
 CSRF_COOKIE_HTTPONLY = True
 CSRF_COOKIE_SECURE = False
 
-# Internationalization
-# https://docs.djangoproject.com/en/1.7/topics/i18n/
 
-LANGUAGE_CODE = 'en-gb'
+# Database
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.environ.get('DB_NAME', 'mtp_api'),
+        'USER': os.environ.get('DB_USERNAME', 'postgres'),
+        'PASSWORD': os.environ.get('DB_PASSWORD', ''),
+        'HOST': os.environ.get('DB_HOST', ''),  # Empty for localhost through domain sockets or '127.0.0.1' for localhost through TCP.
+        'PORT': os.environ.get('DB_PORT', ''),  # Set to empty string for default.
+    }
+}
 
-TIME_ZONE = 'UTC'
 
+# Internationalization
+LANGUAGE_CODE = 'en-gb'
+TIME_ZONE = 'UTC'
```

Comment:
Should we be using `Europe/London`, so we don't have to worry about BST changeovers making the times of actions confusing for our users?